### ISSUE TITLE
htaccess rewriting css path fix-up issue

### ISSUE THREAD

**Issue by DOCaCola** [gobezu/jbetolo#13]
after fixing error #12 i noticed another problem arising with css files.
referenced images in a css file cannot be found after htaccess directory rewriting enabled
so if in a css file that gets merged is a image file path of 'templates/mytemplate/images/toolbar.png' the browser assumes the file to be found in /cache/jbetolo/templates/mytemplate/images/toolbar.png' since the pathes are not fixed by jbetolo correctly with htaccess mode is enabled.

**Comment by realrandal** [comment]
Confirm problem.
But it appears only when DataURL for PNG are enabled. So I think it is right. real problem is that images disappear.

**Comment by gobezu** [comment]
There is now an option called "Map resources inside css absolute" in the CSS section. If you switch between the available ones do you still have problem? If the problem persists I would appreciate if you could provide me a link where the issue appears and I will look at it. Don't hesitate to contact me at info - jproven.com.

**Comment by realrandal** [comment]
Yes, it didn't work at all. I guess it's because of Wrap6 styles (sub-style of main template), so paths to files can be messed.

Try styles in Wrap6 and I think you'll get where is problem.

**Comment by Flimm** [comment]
I experience this issue as well, but I worked around it by enabling "map resources inside css absolute".
